LRQA-29308, LRQA-29666, LRQA-29790 | master

GinsonRen opened the pull request:
https://issues.liferay.com/browse/LRQA-29308
https://issues.liferay.com/browse/LRQA-29666
https://issues.liferay.com/browse/LRQA-29790

liferay-continuous-integration commented:
<h1>Build Started</h1><h4>Job Link:</h4><p><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/25/">test-portal-acceptance-pullrequest(master)</a></p>

liferay-continuous-integration commented:
<h1>Some tests FAILED!</h1><p>Build Time: 2 hours 11 minutes 32 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/228bf39812d5b44ab762fdfe59425818b2c83108">228bf39812d5b44ab762fdfe59425818b2c83108</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/25/">test-portal-acceptance-pullrequest(master)</a></strong></strike></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/264/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk7</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/898/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-hypersonic20-jdk7/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30450/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/265/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/1</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30451/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/10</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32373/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/11</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32374/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/12</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28781/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/13</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/33103/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/14</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32375/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/15</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/267/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/16</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28782/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/17</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30452/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/18</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31284/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/19</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/33102/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/2</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28783/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/20</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30453/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/21</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28784/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/22</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31285/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/23</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30454/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/24</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28785/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/25</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/244/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/26</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/883/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/27</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30455/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/28</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/35649/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/29</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/253/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/3</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/884/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/30</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32376/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/31</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28786/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/32</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30456/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/33</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31286/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/34</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/245/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/35</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32377/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/36</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28780/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/4</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/899/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/5</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/276/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/6</a></li><li><strike><strong><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31283/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/7</a></strong></strike></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32371/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32372/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/9</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/266/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-postgresql94-jdk8/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/33104/">test-portal-acceptance-pullrequest-batch(master)/functional-wildfly10-mariadb100-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28787/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk7</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/31/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/25/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/25/">test-portal-acceptance-pullrequest(master)</a></h5><h6>Job Results:</h6><p>42 Jobs Passed.<br />2 Jobs Failed.</p><pre><code>Completed with the status of FAILURE.</code></pre></li><li><h5 job-result="ABORTED"><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31283/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/7</a></h5><pre><code>Build was aborted</code></pre></li></ol>

brianwulbern commented:
ci:retest

liferay-continuous-integration commented:
<h1>Build Started</h1><h4>Job Link:</h4><p><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest(master)/4/">test-portal-acceptance-pullrequest(master)</a></p>

liferay-continuous-integration commented:
<h1>Some tests FAILED!</h1><p>Build Time: 1 minute 5 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/8b3bcbbe310f6e80c8f807a44300c3632459f3c5">8b3bcbbe310f6e80c8f807a44300c3632459f3c5</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest(master)/4/">test-portal-acceptance-pullrequest(master)</a></strong></strike></li><li><strike><strong><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/3/">test-portal-acceptance-pullrequest-source(master)</a></strong></strike></li></ul><h5>For more details click <a href="https://test-1-12.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest(master)/4/">test-portal-acceptance-pullrequest(master)</a></h5><h6>Job Results:</h6><p>0 Jobs Passed.<br />2 Jobs Failed.</p><pre><code>Completed with the status of FAILURE.</code></pre></li><li><h5 job-result="FAILURE"><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/3/">test-portal-acceptance-pullrequest-source(master)</a></h5><p>Please fix <strong>rebase errors</strong> on <strong><a href="https://github.com/kmaria/liferay-portal/tree/master-qa-16107">kmaria/master-qa-16107</a></strong>.</p><pre><code>     \[exec\] First, rewinding head to replay your work on top of it\.\.\.
     \[exec\] Applying: LRQA\-16107 Add new path for Category selection
     \[exec\] Applying: LRQA\-16107 Extend macro with Categories
     \[exec\] Applying: LRQA\-16107 Add new test case for categories export
     \[exec\] Using index info to reconstruct a base tree\.\.\.
     \[exec\] M	portal\-web/test/functional/com/liferay/portalweb/tests/enduser/wcm/exportimport/ExportImport\.testcase
     \[exec\] Falling back to patching base and 3\-way merge\.\.\.
     \[exec\] Auto\-merging portal\-web/test/functional/com/liferay/portalweb/tests/enduser/wcm/exportimport/ExportImport\.testcase
     \[exec\] CONFLICT \(content\): Merge conflict in portal\-web/test/functional/com/liferay/portalweb/tests/enduser/wcm/exportimport/ExportImport\.testcase
     \[exec\] Failed to merge in the changes\.
     \[exec\] Patch failed at 0003 LRQA\-16107 Add new test case for categories export</code></pre></li></ol>

brianwulbern commented:
@kmaria 

Can you fix the rebase error and force push?  Thanks!

kmaria commented:
@brianwulbern Rebased.

kmaria commented:
ci:retest

liferay-continuous-integration commented:
<h1>Build Started</h1><h4>Job Link:</h4><p><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest(master)/14/">test-portal-acceptance-pullrequest(master)</a></p>

liferay-continuous-integration commented:
<h1>Some tests FAILED!</h1><p>Build Time: 1 hour 23 minutes 26 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/8ed3d93b5b1e6636d04e5c25ae46568ffbb3047e">8ed3d93b5b1e6636d04e5c25ae46568ffbb3047e</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest(master)/14/">test-portal-acceptance-pullrequest(master)</a></strong></strike></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/653/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk7</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/293/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-hypersonic20-jdk7/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/244/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/249/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/1</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/297/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/10</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/251/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/11</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/266/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/12</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/671/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/13</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1258/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/14</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/299/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/15</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/267/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/16</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/249/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/17</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/660/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/18</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/599/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/19</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/260/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/2</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/636/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/20</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1286/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/21</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/269/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/22</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/675/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/23</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/638/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/24</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1259/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/25</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1260/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/26</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/254/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/27</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/300/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/28</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/662/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/29</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/622/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/3</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/303/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/30</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/663/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/31</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/678/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/32</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/276/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/33</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/34</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1262/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/35</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1287/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/36</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/263/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/4</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/250/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/5</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/658/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/6</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/271/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/7</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/241/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/264/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/9</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/642/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-postgresql94-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1263/">test-portal-acceptance-pullrequest-batch(master)/functional-wildfly10-mariadb100-jdk8/0</a></li><li><strike><strong><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/274/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk7</a></strong></strike></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/25/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-7.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest(master)/14/">test-portal-acceptance-pullrequest(master)</a></h5><h6>Job Results:</h6><p>42 Jobs Passed.<br />2 Jobs Failed.</p><pre><code>Completed with the status of FAILURE.</code></pre></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/274/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk7</a></h5><h6>Job Results:</h6><p>54 Tests Passed.<br />1 Test Failed.</p><ol><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=0,label_exp=!master/274//testReport/com.liferay.source.formatter/SourceFormatterTest/testSourceFormatter">SourceFormatterTest.testSourceFormatter</a></li></ol></li></ol>

brianwulbern commented:
SF for MKispal.

* Fixed the source formatter issue

brianwulbern commented:
Pushed to ENagayama.
https://github.com/evannagayama/liferay-portal/pull/699

omarezzine01 committed:
LPS-69462 AUI number validator won't match German patterns

pavel-savinov committed:
LPS-69148 avoid changing path language id in case when current language is not available for the site

hudakl committed:
LPS-69638 In the last-changes-by field, show the modified date

Commit:
LPS-69148 SF

pat270 committed:
LPS-69643 - Move add-panel styles from styled to product-navigation-control-menu-theme-contributor

pat270 committed:
LPS-69643 - Normalize all product-menu form-control styles and fix Add Panel Search Bar alignment

pat270 committed:
LPS-69643 - Product Menu dropdown toggle open should have transparent background in styled theme

Ithildir committed:
LPS-67688 Gradle plugin project files

Ithildir committed:
LPS-67688 Gradle plugin

Ithildir committed:
LPS-67688 Test DBSupportArgs class

Ithildir committed:
LPS-67688 Apply default configuration to DB Support plugin

Ithildir committed:
LPS-67688 Apply DB Support and Service Builder plugins together

Ithildir committed:
LPS-67688 Add Gradle test

Ithildir committed:
LPS-67688 Add readme

Ithildir committed:
LPS-67688 Reuse util method

Ithildir committed:
LPS-67688 Update changelog

Ithildir committed:
LPS-67688 Fix properties caching when coming from JCommander

Ithildir committed:
LPS-67688 Keep the portal-tools.properties file in sync

Ithildir committed:
LPS-67688 Regen Maven plugin descriptor

Ithildir committed:
LPS-67688 Semver portal-tools-db-support

Sharryxue committed:
LRQA-29308 Add pause to allow page to load

marcellustavares committed:
LPS-69711 Add DDMFormFieldValueValidator API

marcellustavares committed:
LPS-69711 Track the DDMFormFieldValueValidators @ DDMFormValuesValidator and invoke them when available

marcellustavares committed:
LPS-69711 Move Option validation to the field types

marcellustavares committed:
LPS-69711 Update tests

marcellustavares committed:
LPS-69711 Make sure DDM service use the latest

marcellustavares committed:
LPS-69711 Add @Reference

gregory-bretall committed:
LPS-69721 Don't reset editor data after adding an image to prevent loss of position in editor

marcellustavares committed:
LPS-69711 SF

marcellustavares committed:
LPS-69711 baseline

jonathanmccann committed:
LPS-69704 Document library conversion links are invalid due to missing 'targetExtension' parameter

Commit:
SYNC-1690 Client-side workaround

Commit:
SYNC-1281 Prep next

Ithildir committed:
LPS-63943 Use HTTP for the private repository if going through our Squid

hhuijser committed:
LPS-69730 Improve behavior for TextFormatter.H and TextFormatter.K

TextFormatter.H: 'FriendlyURLMapper' --> 'friendly url mapper' instead of 'friend u r l mapper'
TextFormatter.K: 'FriendlyURLMapper' --> 'friendly-url-mapper' instead of 'friend-u-r-l-mapper'

hhuijser committed:
LPS-69730 Deprecate TextFormatter.P, we should be using TextFormatter.K now as K is an improvement for P.

hhuijser committed:
LPS-69730 Remove test for deprecated method

hhuijser committed:
LPS-69730 Update usage

hhuijser committed:
LPS-69730 Consolidate logic

hhuijser committed:
LPS-69730 Add tests

hhuijser committed:
LPS-69730 Add test examples to comments

Sharryxue committed:
LRQA-29308 Add time to wait for the success message

hhuijser committed:
LPS-66853 Fix possible NPE

hhuijser committed:
LPS-66853 Fix possible NPE

hhuijser committed:
LPS-66853 If no bnd can be found, it means the language key is missing

ealonso committed:
LPS-69708 Javadoc, copy from FileEntryItemSelectorReturnType

marcellustavares committed:
LPS-69711 Move unit test

shinnlok committed:
SYNC-1704 Missed this

nikki-pru committed:
LRQA-29733 - Test should not run on the release tester because of KB

shuyangzhou committed:
LPS-68471 Partially revert dee25ffad12f719c7f07158634dc73d586c6df49, can not deprecated ParamTag, it is still used by <portlet:param>

Ithildir committed:
LPS-69484 cache

Ithildir committed:
LPS-68289 cache

shuyangzhou committed:
LPS-69751 Create portal properties to configure default init parameters for liferay jasper jsp servlet

shuyangzhou committed:
LPS-69752 Avoid unnecessary runtime string concatenation.

hhuijser committed:
LPS-69754 Variable names cannot be Java keywords

hhuijser committed:
LPS-69754 Add tests for Validator.isVariableName

brianchandotcom committed:
LPS-69751 SF

liuxingye committed:
LRQA-29790 Add pause before clicking publish

brianchandotcom committed:
LPS-69730 prep next, use latest

brianchandotcom committed:
LPS-69730 prep next, use latest

brianchandotcom committed:
LPS-67688

brianchandotcom committed:
LPS-67688 prep next, use latest

brianchandotcom committed:
LPS-69730 cache

brianchandotcom committed:
LPS-69730 prep next, use latest

brianchandotcom committed:
LPS-69730 cache

brianchandotcom committed:
LPS-69730 update build.gradle

brianchandotcom committed:
LPS-69730 update build.gradle

brianchandotcom committed:
LPS-69730 prep next, use latest

brianchandotcom committed:
LPS-69730 proper fix for missing com/liferay/portal/kernel/util/StringBundler

brianchandotcom committed:
LPS-69730 prep next, use latest

brianchandotcom committed:
LPS-69730 Regen

brianchandotcom committed:
LPS-69730 cache

Hanlf committed:
LRQA-29666 Remove type command since LPS-69472

Commit:
LPS-69755 Allow the JSP compiler to use Java 8 constructs

Commit:
LPS-69730 Revert

brianchandotcom committed:
LPS-69730 Recommit

This reverts commit 1182979b8c71ad324fd5be0f3011ae5e2f9cf6e2.

brianchandotcom committed:
LPS-69730 Manual revert one autogenerated code

brianchandotcom committed:
LPS-69730 auto SF

Ithildir committed:
LPS-69730 Move logic to class

Ithildir committed:
LPS-69730 Fix accessor name for LayoutFriendlyURL.layoutFriendlyURLId

shuyangzhou committed:
LPS-69773 Change IncludeTag's logger level to ERROR

nikki-pru committed:
LRQA-29793 Fix path

kenjiheigel committed:
LRQA-29797 Remove 1975 London Theme from release tester

nikki-pru committed:
LRQA-29664 Set variable

michaelhashimoto committed:
LRQA-29795 Fix downloading extra apps

brianchandotcom committed:
LPS-69730 Regen

brianchandotcom committed:
LPS-69730 prep next, use latest

brianchandotcom committed:
LPS-69730 cache

codyhoag committed:
LPS-69708 Edit Javadoc

codyhoag committed:
LPS-69708 Fix Javadoc formatting in similar *ItemSelectorReturnType class, for consistency

codyhoag committed:
LPS-69708 formatJavadoc

Ithildir committed:
LPS-69730 Update changelog

Ithildir committed:
LPS-66853 Fix changelog

liferay-continuous-integration commented:
<h1>All tests PASSED.</h1><p>Build Time: 1 hour 15 minutes 33 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/650dcc942b8d795ddf3254986a222943b8133af6">650dcc942b8d795ddf3254986a222943b8133af6</a></p><h4>Job Summary:</h4><ul><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest(master)/1315/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/67934/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk8</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/67936/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-jboss70-mysql56-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68455/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-mariadb100-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23005/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-mysql56-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/67937/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-oracle121-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68456/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-postgresql94-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/67939/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-sybase160-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/48697/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-weblogic121-mysql56-jdk8/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68895/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-wildfly100-mariadb100-jdk8/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68897/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-hypersonic20-jdk8/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/67690/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68464/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/1</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22700/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/10</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/67694/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/11</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/67948/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/12</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23009/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/13</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/63538/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/14</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/69997/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/15</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68523/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/16</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/69312/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/17</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23011/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/18</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/3565/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/19</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22698/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/2</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/63540/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/20</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/62782/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/21</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/48703/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/22</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/72104/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/23</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/53218/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/24</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68903/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/25</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68465/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/26</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/59989/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/27</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68902/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/28</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68466/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/29</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/67691/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/3</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22940/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/30</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/53219/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/31</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/59991/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/32</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/69998/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/33</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68905/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/34</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22941/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/35</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/67291/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/36</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22704/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/37</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/72107/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/38</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/59992/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/39</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/53216/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/4</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/53220/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/40</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68927/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/41</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68470/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/42</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22697/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/5</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/67944/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/6</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/69308/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/7</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/67946/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22699/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/9</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/62784/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mariadb100-jdk8/0</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/67268/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mariadb100-jdk8/1</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68911/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mysql56-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22709/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mysql56-jdk8/1</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/62788/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-oracle121-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/67962/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-oracle121-jdk8/1</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/67702/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-postgresql94-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/53224/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-postgresql94-jdk8/1</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/67965/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-dist(master)/272/">test-portal-acceptance-pullrequest-dist(master)/jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/316/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-3.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1315/jenkins-report.html">here</a>.</h5><h5>For upstream results, click <a href="http://test-1-1/job/test-portal-acceptance-upstream(master)/">here</a>.</h5>

brianwulbern commented:
@liuxingye if https://issues.liferay.com/browse/LRQA-29790 is still a flaky issue within 20 test runs, please resend me a pull for it, thanks!
